UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -1034,4 +1035,24 @@ public void CmdResetMovementForSelf()
 		playerScript.playerMove.ResetEverything();
 		playerScript.playerMove.ResetLocationOnClients();
 	}
+
+	[Command]
+	public void CmdJoinFaith(string faith)
+	{
+		playerScript.JoinReligion(faith);
```

Comment:
iis it okay for the player to join any faith at any time?

Reply:
It will leave the previous faith, if any, before joining the new one.

Reply:
Considering that in the future there will be the ability for players to join faiths at the start or via other interactions, yeah I think its fine.